src/HW_models/NRF_CLOCK.h

```diff
@@ -22,6 +22,7 @@ void nrf_clock_TASKS_HFCLKSTART();
 void nrf_clock_reqw_sideeffects_INTENSET();
 void nrf_clock_reqw_sideeffects_INTENCLR();
 void nrf_clock_reqw_sideeffects_TASKS_LFCLKSTART();
+void nrf_clock_reqw_sideeffects_TASK_LFCLKSTOP();
```

Comment:
I think this should be called `nrf_clock_reqw_sideeffects_TASKS_LFCLKSTOP` to be in line with the other functions.

Reply:
I agree with @wopu-ot (and sorry for the typo in my original comment)